core/templates/dev/head/pages/exploration_editor/ExplorationSaveAndPublishButtonsDirective.js

```diff
@@ -117,6 +117,13 @@ oppia.directive('explorationSaveAndPublishButtons', [
                 $scope.loadingDotsAreShown = false;
               });
           };
+
+          $scope.showTooltip = function() {
```

Comment:
Could the function name also include some details about which Tooltip? showTooltip sounds a bit too generic.

Reply:
Done